README.md

```diff
@@ -182,7 +190,7 @@ provide APIs to simplify this communication.
 
 ### Extend Triton
 
-[Triton Inference Server's archicture](docs/architecture.md) is specifically 
+[Triton Inference Server's architecture](docs/architecture.md) is specifically 
 designed for modularity and flexibility
```

Comment:
Add trailing "."

Reply:
No, none of the other points have that. Please keep review to only lines changed.